src/track/beatgrid.cpp

```diff
@@ -365,4 +365,13 @@ void BeatGrid::setBpm(double dBpm) {
     emit updated();
 }
 
+void BeatGrid::round() {
+    using ::round;
```

Comment:
This using directive is Prefer to prefix it explicitly once, i.e. by invoking ::round(). Not necessary after this method has been renamed.

Suggestion:
```
double roundedBpm = round(getBpm());
...
```
Using intermediate, mutable local variables is not necessary and makes the code only harder to read.